src/js/pagination/specs/paging-collection-spec.js

```diff
@@ -345,25 +350,27 @@ define(['jquery',
 
                     it('triggers an error event when the requested page is out of range',
                         AjaxHelpers.withFakeRequests(function(requests) {
-                            var errorTriggered = false;
+                            // TODO: Re-enable expectations and remove eslint ignores
```

Comment:
I fixed it! There's a bad bug in `sinon@1.17.4`, downgrading it to the previous version allows these tests to be uncommented and pass. See https://github.com/sinonjs/sinon/issues/1040

Also fixes the coverage issues.

😌